Makefile

```diff
@@ -13,7 +13,17 @@ endif
 
 CPPFLAGS += -Ilibvterm-0.1.3/include -DGITHASH=$(GITHASH)
 CFLAGS   += -Wall -falign-labels=8
-CXXFLAGS += -Wall -falign-labels=8 -std=gnu++17 -fpermissive
+CXXFLAGS += -Wall -falign-labels=8 -fpermissive
+
+# Attempt to automatically drop -static-libstdc++ when using the Nickel TC...
+ifeq ($(shell PATH='$(PATH)' $(CROSS_TC)-gcc -dumpmachine 2>/dev/null), arm-nickel-linux-gnueabihf)
+	STATIC_STL_FLAG:=
+	# What can I say, it's old...
```

Comment:
Sounds fun :)

Reply:
To be fair, the Kindle rootfs is also ridiculously old (and softfloat), the PocketBoot rootfs has only recently been updated, and it's not terribly fresh either.

reMarkable seem to care a bit more about these things, they're using a fairly recent rootfs, as well as slightly less old kernels (given the NXP constraints).